Find a functionally equivalent code:android.app.admin.DevicePolicyManager.getActiveAdmins:COMMENT
Method Modifier: public      
Comment:/**
 * Return a list of all currently active device administrator's component
 * names.  Note that if there are no administrators than null may be
 * returned.
 */

@@ -1,10 +1,3 @@
 {
-    if (mService != null) {
-        try {
-            return mService.getActiveAdmins(UserHandle.myUserId());
-        } catch (RemoteException e) {
-            Log.w(TAG, "Failed talking with device policy service", e);
-        }
-    }
-    return null;
+    return getActiveAdminsAsUser(UserHandle.myUserId());
 }

